src/app/views/sap/create_mission.php

```diff
@@ -14,13 +14,12 @@
                 <h1>Create new mission</h1>
                 <?php
                 if (isset($errors) && count($errors) !== 0) {
-                    foreach ($errors as $error) {
-                        echo "<p style=\"color:red\">$error</p>";
-                    }
-                } else if (isset($result) && $result === false) {
-                    echo "<p style=\"color:red\">Something went wrong. :/</p>";
-                }
-                ?>
+                    foreach ($errors as $error) { ?>
```

Comment:
I agree. However, when it's in between another block of PHP, that makes things weird. @hharchani?